autocti/model/result.py

```diff
@@ -2,11 +2,23 @@
 
 
 class Result(result.Result):
-    def __init__(self, samples_summary, paths=None, samples=None, analysis=None, search_internal = None):
+    def __init__(
+        self,
+        samples_summary,
+        paths=None,
+        samples=None,
+        analysis=None,
+        search_internal=None,
+    ):
         """
         The result of a phase
         """
-        super().__init__(samples_summary=samples_summary, paths=paths, samples=samples, search_internal=search_internal)
+        super().__init__(
+            samples_summary=samples_summary,
+            paths=paths,
+            samples=samples,
+            search_internal=search_internal,
+        )
 
         self.analysis = analysis

```

Comment:
If we allowed results to take additional kwargs in general you would be able to pass a generic result an analysis and would not need to define this class

Reply:
Noted, will make the API change in the future.